docs/release/release_0_4_3.md

```diff
@@ -0,0 +1,120 @@
+
+In this release we also seperate out more of the Qt functionality from napari making it easier to run headless (#2039, #2055). We also add a `napari.run` method to then launch the napari application (#2056).
+
+We've also made good progress on our experimental support for an octree system for rendering large 2D multiscale images. You can try this functionality setting `NAPARI_ASYNC=1` as an environment variable.
+
+
+## New Features
+- Add support for function widgets (#1856)
+
+
+## Improvements
+- Use evented dataclass for dims (#1917)
+- Replace layer dims with private named tuple (#1919)
+- Add information about screen resolution (#1957)
+- Add asdict and update to evented dataclass (#1966)
+- async-30: ChunkLoader Stats (#1972)
+- async-31: Fix Monitor Feature Toggle (#1975)
+- octree: Fix float64 and check downscale ratio (#1976)
+- async-32: Faster Octree Rendering (#1977)
+- async-33: Cleanup QtPoll and Tiled Visuals (#1980)
+- async-34: Octree Performance (#1989)
+- async-35: OctreeDisplayOptions (#1991)
+- async-36: New OctreeChunkLoader (#1992)
+- async-37: Multilevel Rendering (#1995)
+- async-38: Better Multilevel Rendering (#1997)
+- async-39: Better Config and Remove Qt Code (#1999)
+- Improve label paint lag (#2000)
+- async-40: Shared Memory Resources (#2004)
+- async-41: Better Octree Performance (#2010)
+- Start magicgui apps too (#2016)
+- async-42: New LoaderPool and remove ChunkKey (#2025)
+- Two small path improvements (#2030)
+- Cleanup dockwidget removal (#2036)
+- Prepare for evented viewer (#2038)
+- Add a MousemapProvider (#2040)
+- Cleanup warnings in tests (#2041)
+- async-43: New LoaderPoolGroup (#2049)
+- Complete qt isolation (#2055)
+- Create QApplication on demand in viewer.Window, add napari.run function (#2056)
+- Remove global app logic from Window init (#2065)
+- async-45: Docs and Cleanup (#2067)
+
+## Bug Fixes
+- Fix append and remove from layerlist (#1955)
+- Change label tooltip checkbox (#1978)
+- Fix cursor size (#1983)
+- Fix layer removing by disconnecting each event individually (#1984)
+- Compatibility with magicgui v0.2.0 (#1994)
+- Skip rounding of numbers when comparing data slice (#2017)
+- Fix InitVar problem (#2023)
+- Fix channel_axis for affine transforms in add_image (#2026)
+- Fix typos in theme event and vispy camera (#2034)
+- Fix magicgui test funcs (#2044)
+- Fix magicwidget.native detection of "empty" widgets (#2046)
+- async-44: Fix Pixel Shift Bug (#2052)
+- Fix missing console widget (#2063)
+
+
+## API Changes
+- Drop layer.shape (#1990)
+- Drop image layer.shape (#2002)
+- Move keymap handling off viewer (#2003)
+- Drop scale, axes background color, connect directly to viewer (#2037)
```

Comment:
Can you add notes here about what the new API should be?

Reply:
done